src/views/Colors/ThemeCircle.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="theme-card">
+    <div class="circle-border" :class="{ 'no-border': !isCurrentTheme }">
+      <div class="circle" :style="{ background: gradientString }"></div>
+    </div>
+    <div class="name">
+      {{ theme.name.toUpperCase().substring(0, 1) + theme.name.substring(1) }}
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  props: {
+    theme: { type: Object, required: true },
+    isCurrentTheme: { type: Boolean, required: true },
+  },
+  data() {
+    return {
+      colors: [],
+    };
+  },
```

Comment:
can we get rid of this now? since we changed the below `gradientString` to not use `colors` anymore